Description: add missing "fs_utf8" feature dependency from cap-std
Author: Ananthu C V <weepingclown@disroot.org>
Last-Update: 2024-11-02

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -44,2 +44,3 @@
 [dependencies.cap-std]
+features = ["fs_utf8"]
 version = "^3.4.4"
